packages/bento-design-system/src/util/atoms.ts

```diff
@@ -2,6 +2,7 @@ import { vars } from "../vars.css";
 
 const radius = {
   ...vars.radius,
+  0: 0,
```

Comment:
added here and not in the theme, otherwise it would be a breaking change.

(@veej heads up, I'm planning to remove a bunch of variables from the theme since I don't think they belong there, radius is a good example)